test.js

```diff
@@ -52,6 +52,69 @@ test('margin option', t => {
       │               │
       │               │
       └───────────────┘
-      
+
     `);
 });
+
+test('borderType "double"', t => {
+	compare(t, fn('foo', {borderType: 'double'}), `
+╔═══╗
+║foo║
+╚═══╝
```

Comment:
Would be useful if the different options in the readme had a preview like this one for each style. Would make it easier to choose.